src/libs/actions/Report.js

```diff
@@ -1073,27 +1054,18 @@ function subscribeToNewActionEvent(reportID, callback) {
 }
 
 /**
+ * Sends callbacks and notifications for new report actions.
+ *
  * @param {String} reportID
  * @param {Object} action
  */
-function viewNewReportAction(reportID, action) {
+function sendNewReportActionNotification(reportID, action) {
```

Comment:
I would change this to `presentReportActionNotification()` if we are struggling to get the function name to make sense because of the callback then it might be a good idea to just remove the callback logic out of this and have the subscriber callback get called here instead:

```js
            presentReportActionNotification(reportID, action);

            // Notify the ReportActionsView that a new comment has arrived
            if (reportID === newActionSubscriber.reportID) {
                newActionSubscriber.callback(isFromCurrentUser, action.reportActionID);
            }
            handledReportActions[reportID] = handledReportActions[reportID] || {};
            handledReportActions[reportID][action.sequenceNumber] = true;
        });
```

Reply:
Ah I like that. I'll do it that way.